tensorflow/core/kernels/unique_op_gpu.cu.h

```diff
@@ -331,7 +332,11 @@ class UniqueOpGPU : public AsyncOpKernel {
       const GPUDevice& device = context->eigen_gpu_device();
       int64 uniq_size = (*last_idx_host.data()) + 1;
 
+#if GOOGLE_CUDA
       se::cuda::ScopedActivateExecutorContext scoped_activation{
+#else
+      se::gpu::ScopedActivateExecutorContext scoped_activation{
```

Comment:
Can we use se::gpu:: for both?

Reply:
Yea, this was probably overly careful. I'll test this as se::gpu on a NV machine to make sure.